machine1/dbus.go

```diff
@@ -75,6 +77,64 @@ func (c *Conn) initConnection() error {
 	return nil
 }
 
+func (c *Conn) getPath(method string, args ...interface{}) (dbus.ObjectPath, error) {
+	result := c.object.Call(fmt.Sprintf("%s%s", dbusInterface, method), 0, args...)
```

Comment:
This was just based on `dbusInterface+".Method"` that was already there.  I agree having it be `fmt.Sprintf("%s.%s", dbusinterface, method)` is better.